doc/examples/applications/plot_3d_structure_tensor.py

```diff
@@ -0,0 +1,164 @@
+"""
+==========================================
+Estimate isotropy in a 3D microscopy image
+==========================================
+
+In this tutorial, we compute the structure tensor of a 3D image.
+For a general introduction to 3D image processing, please refer to
+:ref:`sphx_glr_auto_examples_applications_plot_3d_image_processing.py`.
+The data we use here are sampled from an image of kidney tissue by Genevieve
+Buckley in confocal fluorescence microscopy (more details at [1]_ under
+``kidney-tissue-fluorescence.tif``).
+
+.. [1] https://gitlab.com/scikit-image/data/#data
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import plotly.express as px
+from skimage import (
+    data, feature
+)
+
+
+#####################################################################
+# Load image
+# ==========
+# This biomedical image is available through `scikit-image`'s data registry.
+
+data = data.kidney()
+
+#####################################################################
+# What exactly are the shape and size of our 3D multichannel image?
+
+print(f'number of dimensions: {data.ndim}')
+print(f'shape: {data.shape}')
+print(f'dtype: {data.dtype}')
+
+#####################################################################
+# For the purposes of this tutorial, we shall consider only the second colour
+# channel, which leaves us with a 3D single-channel image. What is the range
+# of values?
+
+n_plane, n_Y, n_X, n_chan = data.shape
+v_min, v_max = data[:, :, :, 1].min(), data[:, :, :, 1].max()
+print(f'range: ({v_min}, {v_max})')
+
+#####################################################################
+# Let us visualize the middle slice of our 3D image.
+
+px.imshow(
+    data[n_plane // 2, :, :, 1],
+    zmin=v_min,
+    zmax=v_max,
+    labels={'x': 'Y', 'y': 'X', 'color': 'intensity'}
+)
+
+#####################################################################
+# Let us pick a specific region, which shows relative X-Y isotropy. In
+# contrast, the gradient is quite different (and, for that matter, weak) along
+# Z.
+
+sample = data[5:13, 380:410, 370:400, 1]
+
+
+def display(im3d, cmap='gray', step=3):
```

Comment:
Defining the `display` function is not necessary since it is used only once...

Reply:
You're right. It doesn't even work well for readers to reuse (say, changing the `step` value), because the number of rows and columns are hard-coded for the subplots! Fixed in 808b9a7f2ba129556aad6d68f367903afd647a72.

PS: thanks for suggesting `enumerate()` to get a counter in the loop! :woman_facepalming: